code/modules/mob/living/simple_animal/animals/giant_spider.dm

```diff
@@ -95,7 +99,107 @@
 	old_x = -16
 	old_y = -16
 
+/mob/living/simple_animal/hostile/giant_spider/webslinger
+	desc = "Furry and green, it makes you shudder to look at it. This one has brilliant green eyes, and a cloak of web."
+	icon_state = "webslinger"
+	icon_living = "webslinger"
+	icon_dead = "webslinger_dead"
+
+	maxHealth = 90
+	health = 90
+	taser_kill = 0
```

Comment:
Why taser_kill = 0?

Reply:
Most of these are because 'Mechoid is a donk and made a frame for spider forms that included it, because of pit and sparky'

Reply:
My love of automation wounds me again.